content/en/dev/setup.md

```diff
@@ -49,31 +49,41 @@ You can follow the [pre-requisites instructions from the production guide]({{<re
 Run the following commands in the project directory:
 
 ```sh
+bundle config set --local path vendor/bundle
 bundle install
-yarn install
+yarn
 ```
 
+{{<hint style="info">}}
+On some operating systems, such as Debian and Fedora, `yarn` has been renamed to `yarnpkg`. On these systems, run `yarnpkg` instead of `yarn`.
+{{</hint>}}
+
 In the development environment, Mastodon will use PostgreSQL as the currently signed-in Linux user using the `ident` method. Ensure that you have created a Postgres user and database for your current signed-in user:
 
 ```sh
-sudo -u postgres createuser $YOUR_USERNAME_HERE --createdb
+sudo -u postgres createuser $USER --createdb
 ```
 
-You can now create the databases `mastodon_development` and `mastodon_test`, load the schema into them, and create seed data defined in `db/seeds/` into `mastodon_development`.
+{{<hint style="info">}}
+If the above fails, you may have to run the following to initialize PostgreSQL:
 
 ```sh
-rails db:setup
+sudo postgresql-setup --initdb
```

Comment:
this seems distro-specific, as i've not seen a `postgresql-setup` command in e.g. arch. from my very quick searching, it might be a fedora thing?

Reply:
I believe this should be:
```
RAILS_ENV=development bundle exec rails db:setup
```
(Analogous to the command [here](https://docs.joinmastodon.org/admin/install/#generating-a-configuration).)

Reply:
@trwnh Oh maybe, I do only remember having to do this on Fedora. Perhaps the documentation should not detail how to set up PostgreSQL and simply say "refer to your distribution's documentation" or something, since it seems like the distributions set things up differently

Reply:
@nuztalgia See line 78 :smile: The Git diff is messed up